contents/blog/leo-anthias-interview.md

```diff
@@ -10,31 +10,41 @@ featuredImage: ../images/blog/leo-anthias-interview.png
 
 > It's really nice to just focus on one thing. For the last two or three days, I've really only focused on our onboarding flows for guides and tutorials for new users. And being able to just focus on that one thing has meant that probably I've gotten a lot more done than if I tried to do that while at the same time running a hiring process, while at the same time having a fundraising process, whatever. Even though all of those things are of equal importance, it's really key to just pick one of them at a time.
 
-After returning to the UK, Leo co-founded another company: a data science model deployment platform. Their goal was to make it easy for engineers to publish a model to the cloud, scale it, and integrate it. Part of that service was a library they built for their customers, which they eventually spun out into a separate product - Datapane. 
+After returning to the UK, Leo co-founded another company: a data science model deployment platform. Their goal was to make it easy for engineers to publish a model to the cloud, scale it, and integrate it. Part of that service was a library they built for their customers, which they eventually spun out into a separate product - Datapane.
 
-Datapane’s birth was inspired by data scientists leaning more on no-code tools and plugins for analysis, visualization, and collaboration. Being locked into proprietary software ecosystems makes it hard to publish new research with the world, so Datapane aims to simplify going from coding data in Python to sharing snippets on Medium or in PowerPoint decks. Leo and his team thus optimize for the number of people building and sharing reports.
+![image](https://user-images.githubusercontent.com/40317687/127475264-c5910fcf-4194-440e-be11-1f72e6f6a093.png)
 
-Datapane uses Django as a base, Alpine JS for the frontend, Tailwind CSS for the design, React for the interactive slides, and Kubernetes for handling orchestration, workload execution, and other cloud-based tasks. One of Datapane’s best features is being able to create reports in your browser (using a Markdown editor) without needing to go through Python libraries. Leo and his team are also looking to build a mobile app for report viewing and sharing in the near future.
+Datapane’s birth was inspired by data scientists who didn't want to use drag-and-drop tools (like Tableau) and actually wanted to write code for analysis, visualization, and collaboration. Being locked into proprietary software ecosystems makes it hard to publish new research with the world, so Datapane aims to simplify going from coding data in Python to sharing snippets on Medium or in PowerPoint decks. Leo and his team thus optimize for the number of people building and sharing reports.
```

Comment:
"Datapane is an open-source library which makes it easy for data scientists to create interactive data science reports using Python which they can share with their team or community."

Reply:
Added